django_rq/templates/django_rq/job_detail.html

```diff
@@ -89,6 +91,19 @@
             </div>
         </div>
 
+        {% if job.dependency %}
+        <div class="form-row">
+            <div>
+                <label class="required">Dependency:</label>
+                <div class="data">
+                  <a href = "{% url 'rq_job_detail' queue_index job.dependency.id %}">
+                    {{ job.dependency.func_name }}
```

Comment:
I think we should also display the `job.id` here, something like this would be great `{{ job.dependency.id }} ({{ job.dependency.func_name }}) `